src/Security/CheckGroups/CheckGroupServerConfig.js

```diff
@@ -50,14 +51,25 @@ class CheckGroupServerConfig extends CheckGroup {
       }),
       new Check({
         title: 'Client class creation disabled',
-        warning: 'Attackers are allowed to create new classes without restriction and flood the database.',
-        solution: 'Change Parse Server configuration to \'allowClientClassCreation: false\'.',
+        warning:
+          'Attackers are allowed to create new classes without restriction and flood the database.',
+        solution: "Change Parse Server configuration to 'allowClientClassCreation: false'.",
         check: () => {
           if (config.allowClientClassCreation || config.allowClientClassCreation == null) {
             throw 1;
           }
         },
       }),
+      new Check({
+        title: 'Public Users on signup',
```

Comment:
This should be adapted to follow the [wording guideline](https://github.com/parse-community/parse-server/blob/master/CONTRIBUTING.md#wording-guideline). Please let me know if the guideline is unclear or needs other improvements.

Reply:
My mistake Manuel, I hadn't realised you'd created the wording guideline. The guideline is perfect.